Makefile

```diff
@@ -100,46 +100,46 @@ build:
 ###                                 Localnet                                ###
 ###############################################################################
 localnet-keys:
-	. test/localjuno/scripts/add_keys.sh
+	. tests/localjuno/scripts/add_keys.sh
 
 localnet-init: localnet-clean localnet-build
 
 localnet-build:  
-	@chmod -R +x test/localjuno/
-	@DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f test/localjuno/docker-compose.yml build
+	@chmod -R +x tests/localjuno/
+	@DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f tests/localjuno/docker-compose.yml build
 
 localnet-start:  
-	@STATE="" docker-compose -f test/localjuno/docker-compose.yml up
+	@STATE="" docker-compose -f tests/localjuno/docker-compose.yml up
 
 localnet-start-with-state:	
-	@STATE=-s docker-compose -f test/localjuno/docker-compose.yml up
+	@STATE=-s docker-compose -f tests/localjuno/docker-compose.yml up
 
 localnet-startd:
-	@STATE="" docker-compose -f test/localjuno/docker-compose.yml up -d
+	@STATE="" docker-compose -f tests/localjuno/docker-compose.yml up -d
 
 localnet-startd-with-state:
-	@STATE=-s docker-compose -f test/localjuno/docker-compose.yml up -d
+	@STATE=-s docker-compose -f tests/localjuno/docker-compose.yml up -d
 
 localnet-stop:
-	@STATE="" docker-compose -f test/localjuno/docker-compose.yml down
+	@STATE="" docker-compose -f tests/localjuno/docker-compose.yml down
 
 localnet-clean:
-	@rm -rfI $(HOME)/.juno/
+	@rm -rfI $(HOME)/.juno/Dockerfile
```

Comment:
This should just be `@rm -rfI $(HOME)/.juno/` right? we don't store a Dockerfile in the .juno folder anywhere I can see.

Also wouldn't this need to be sudo? Since Docker runs as privileged 
```
(130) reece@pbcupsd [~/Desktop/Programming/Go/juno]
(local_juno) -> $ rm -rf ~/.juno/
rm: cannot remove '/home/reece/.juno/config/priv_validator_key.json': Permission denied
rm: cannot remove '/home/reece/.juno/config/node_key.json': Permission denied
rm: cannot remove '/home/reece/.juno/config/addrbook.json': Permission denied
rm: cannot remove '/home/reece/.juno/config/client.toml': Permission denied
rm: cannot remove '/home/reece/.juno/config/genesis.json': Permission denied
rm: cannot remove '/home/reece/.juno/config/config.toml': Permission denied
rm: cannot remove '/home/reece/.juno/config/app.toml': Permission denied
rm: cannot remove '/home/reece/.juno/data': Permission denied
```

Reply:
Yea!! You right. Don't know why I added it, maybe by accident, I'll fix it now